docs/domains.rst

```diff
@@ -220,7 +220,17 @@ file like this:
   intents:
     - greet: {use_entities: false}
 
-This means that entities for those intents will be unfeaturized and therefore
+To ignore some entities or explicitly take only certain entities
+into account you can use this syntax:
+
+.. code-block:: yaml
+
+  intents:
+    - greet: {use_entities: false, include_entities: [name]}
```

Comment:
If one entity is explicitly included, we don't need the `use_entities: false` right?

Reply:
Currently `use_entities` defaults to true, no matter whether `include_entities` or `exclude_entities` are specified. Thus, if we want to include only one specific entity we need to exclude all by default, by setting `use_entities: false` and then explicitly include the wanted entity. Otherwise, all entities will get included.

Reply:
Mhm, how about changing it so that `include_entities: ["default"]` automatically means that all others are excluded? This seems more intuitive to me, than `use_entities: false` + `include_entities`.

Reply:
I'll add that

Reply:
Done. I also removed some redundant code. There were 3 places that set a default value for use_entities. The only place in the code where the parameter is actually used is the part that decides whether to featurize it. Thus I removed it from the other 2 places.